src/OSPSuite.Core/Domain/EntitySource.cs

```diff
@@ -42,20 +43,30 @@ public EntitySource()
       {
       }
 
-      public EntitySource(EntitySource originalSource) : this(originalSource.BuildingBlockId, originalSource.SourceType, originalSource.SourceId, originalSource.Source)
+      internal EntitySource(EntitySource originalSource) : this(originalSource.BuildingBlockId, originalSource.SourceType, originalSource.SourceId, originalSource.Source)
       {
          EntityPath = originalSource.EntityPath;
       }
 
-      public EntitySource(string buildingBlockId, string sourceType, string sourceId, IEntity source)
+      internal EntitySource(string buildingBlockId, string sourceType, string sourceId, IEntity source)
       {
          BuildingBlockId = buildingBlockId;
          SourceType = sourceType;
          SourceId = sourceId;
          Source = source;
       }
 
-      public EntitySource Clone() => new EntitySource(this);
+      /// <summary>
+      ///    Returns a clone of the object without the reference to the source
+      /// </summary>
+      /// <returns></returns>
+      public EntitySource Clone()
```

Comment:
When I clone (e.g. usage in MoBi), I don't want to save the source to avoid leaks.